### ISSUE TITLE
Reduce common texture size and migrate text labels to independent hi-res decals

### ISSUE THREAD

**Issue by sumghai** [sumghai/FusTek_StationParts#52]
Up until this point, the newly-remapped common textures and normal maps were a whopping 4096 x 4096 px - the original intention was that this resolution was the minimum required to make the following text labels legible:
 - [x] **CAUTION - BAY DOORS MAY MOVE WITHOUT WARNING** on Warehouse / Payload Bay
 - [x] **AIRLOCK ACCESS** on Kuest and Kuest Legacy Airlocks
 - [x] **VISITING SPACECRAFT END - INSTALL DOCKING PORT BEFORE FLIGHT** on Kirs Docking Module

However, having learned new texturing techniques such as using translucent alpha textures to make hi-res "floating" decals (that have been successfully implemented on the Karmony hatches and Warehouse control panels), I think it's reasonable to reduce the common textures and normal maps to 2048 x 2048 px, while migrating the aforementioned text labels to decals.

Lowering the resolution of the main textures should ease RAM loading significantly, hopefully only at a slight loss in quality.

**Comment by InsaneDruid** [comment]
Especially since KSP 1.0 will have proper DDS support I don't think that such an reduction will lead to an significant performance gain anymore. But it will reduce texture fidelity, especially as the most parts only use a fraction of the size of the texture. For users that do want smaller textures ksp already has its texture size-slider, and (at the moment) both ATM and dds-loader can be configured to reduce texture size/drop the biggest mipmap, too. So long story short: I would keep the size, so that players who want more fidelity can have them, while players beein desparate can adjust the size ingame wihout problems.

**Comment by sumghai** [comment]
@InsaneDruid 

I understand what you mean. I made this decision because I am planning on including nothke's re-licensed textures into the parts pack as well, and people were complaining that they were hitting their memory ceiling even at the lowest texture settings and with ATM / DDS loader.

------------

I've turned all the low-res text into hi-res decals located in the bottom-left transparent corner of the same common texture atlas, so that we don't need to load yet another separate texture for this purpose. This would also allow alt-texturers (such as @InsaneDruid yourself :D) to easily devise your own text labels, or even omit altogether them if they don't fit your particular style.

Now, to wait for user feedback... :)
